### PR TITLE
kba like repo setup

### PR DESCRIPTION
**Pull request by juliangieseke**
this repo should have the same setup as kba since its also managed by konvoy in cluster.yaml file. ~no catalog like system is in place~ Konvoy doesnt support selection of addon version, only via `configVersion` branch/tag.

main reasons for aligning the two repos is the abovementioned and ease of use in terms of documentation/rememebering how to deal with this repo.

### CODE REVIEW

**Comment by shaneutt** [comment]
> this repo should have the same setup as kba since its also managed by konvoy in cluster.yaml file. no catalog like system is in place

Just as a bit of clarification: it's not quite accurate to say "no catalog system is in place", as Konvoy itself now uses the catalog in order to navigate these repositories.

**Comment by juliangieseke** [comment]
yeah, you're right, I was referring to how we use it and what `kubeaddons-enterprise` is, where you can select individual versions per addon in kommander UI. thats not in place for this repo, here its like `kubernetes-base-addons` where you select a tag/branch via `configVersion` in cluster.yaml.

File: README.md [hunk 1]
```diff
@@ -2,18 +2,10 @@
 
 This is a [kubernetes addon](https://github.com/mesosphere/kubeaddons) repository which contains [Kommander](https://github.com/mesosphere/kommander) related addon applications.
 
-# Updating Kommander Addon
+# Using and releasing `kubeaddons-kommander` repo
 
-**This section is only intended for short term guideance and might not be/stay 100% accurate, please get rid of whatever you see that seems wrong.**
-**this repo follows the same rules as https://github.com/mesosphere/kubernetes-base-addons - so please check [readme](https://github.com/mesosphere/kubernetes-base-addons/blob/master/README.md) and [contributing](https://github.com/mesosphere/kubernetes-base-addons/blob/master/CONTRIBUTING.md) files over there**
+This repository follows the exact same rules that [`kubernetes-base-addons`](https://github.com/mesosphere/kubernetes-base-addons) does, please check documentation over there:
```
Comment (drewvanstone) [inline README.md]:
```suggestion
This repository follows the exact same rules that [`kubernetes-base-addons`](https://github.com/mesosphere/kubernetes-base-addons) does, please check documentation over there.
```

File: README.md [hunk 2]
```diff
@@ -2,18 +2,10 @@
 
 This is a [kubernetes addon](https://github.com/mesosphere/kubeaddons) repository which contains [Kommander](https://github.com/mesosphere/kommander) related addon applications.
 
-# Updating Kommander Addon
+# Using and releasing `kubeaddons-kommander` repo
```
Comment (drewvanstone) [inline README.md]:
```suggestion
# Using and Releasing `kubeaddons-kommander` Repository
```

Comment (drewvanstone) [reply README.md]: This heading should be title case

**Pull request merged**